moe.karaokes.mugen.app.yml

```diff
@@ -0,0 +1,93 @@
+app-id: moe.karaokes.mugen.app
```

Comment:
I have already expressed my concern about the app-id. (the fact that it ends in `.app` just reinforce it)

Reply:
That's because there's an app and a server for KM. We don't plan on releaseing the server via flatpak but we wanted to differentiate it. If that's a big issue, we can remove the `.app`

We're discovering flathub and how to build for it, sorry for the mistakes.